.github/workflows/release-temporal.yml

```diff
@@ -19,40 +19,20 @@ jobs:
   retag-and-release:
     name: "Re-tag and release images"
     runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: bash
 
     steps:
-      - name: Calc image tag from commit sha
-        id: calc_args
-        env:
-          COMMIT: ${{ github.event.inputs.commit }}
-        run: |
-          echo "image_tag=sha-${COMMIT:0:7}" >> $GITHUB_OUTPUT
-
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'src/go.mod'
       - name: Copy images
         env:
+          COMMIT: ${{ github.event.inputs.commit }}
+          TAG: ${{ github.event.inputs.tag }}
```

Comment:
I would use command line arguments instead of `env` with go programm.

Reply:
Good idea. I'll update that in the following PR.